commit 3bd2706c910fd328e4ab96ae0aabdcd7c4a90fbf
Author: Sai Krishna Potthuri <sai.krishna.potthuri@amd.com>
Date:   Wed Jan 4 14:15:12 2023 +0530

    EDAC/zynqmp: Add EDAC support for Xilinx ZynqMP OCM
    
    Add EDAC support for Xilinx ZynqMP OCM Controller, so this driver reports CE and
    UE errors upon interrupt generation. Also add debugfs files for error injection.
    
    On Xilinx ZynqMP platform, both OCM Controller driver(zynqmp_edac) and DDR
    Memory Controller driver(synopsys_edac) co-exist which means both can be loaded
    at a time. This scenario is tested on Xilinx ZynqMP platform.
    
    Fix following issue reported by the robot:
      "MAINTAINERS references a file that doesn't exist:
      Documentation/devicetree/bindings/edac/xlnx,zynqmp-ocmc.yaml"
    
      [ bp:
        - Massage commit message
        - s/EDAC_ZYNQMP_OCM/EDAC_ZYNQMP/
        - Touchups
          ]
    
    Reported-by: kernel test robot <lkp@intel.com>
    Co-developed-by: Shubhrajyoti Datta <shubhrajyoti.datta@amd.com>
    Signed-off-by: Shubhrajyoti Datta <shubhrajyoti.datta@amd.com>
    Signed-off-by: Sai Krishna Potthuri <sai.krishna.potthuri@amd.com>
    Signed-off-by: Borislav Petkov (AMD) <bp@alien8.de>
    Link: https://lore.kernel.org/r/20230104084512.1855243-3-sai.krishna.potthuri@amd.com

diff --git a/drivers/edac/Makefile b/drivers/edac/Makefile
index 2d1641a27a28..9b025c5b3061 100644
--- a/drivers/edac/Makefile
+++ b/drivers/edac/Makefile
@@ -84,3 +84,4 @@ obj-$(CONFIG_EDAC_QCOM)			+= qcom_edac.o
 obj-$(CONFIG_EDAC_ASPEED)		+= aspeed_edac.o
 obj-$(CONFIG_EDAC_BLUEFIELD)		+= bluefield_edac.o
 obj-$(CONFIG_EDAC_DMC520)		+= dmc520_edac.o
+obj-$(CONFIG_EDAC_ZYNQMP)		+= zynqmp_edac.o